apps/researcher/src/messages/en/messages.json

```diff
@@ -192,7 +192,7 @@
   "Gallery": {
     "resetZoomButton": "Reset",
     "license": "License",
-    "noLicense": "This image has been published by {organizationName} without a license - for more information please contact {organizationName}."
+    "noLicense": "We could not find a license for this image - for more information please contact {organizationName}."
```

Comment:
To be sure: who is 'we' - and is that clear to users? Alternatively, use a phrase such as 'This image has no license. For more information please contact...'?

Reply:
I don't think 'This image has no license' is nuanced enough. I can change it to "No image found for this image".